File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/non-plagiarized/14/Soal4.kt
package IR

import java.text.DecimalFormat

/**
 *
 * @author 4900D0A19B6894A4A514E9FF3AFCC2C0
 */
fun main() {
    val arr = Array(11) { IntArray(2) }
    val df = DecimalFormat("#.###")
    for (i in 0..10) {
        if (i == 0) {
            println("Miles \t Kilometers")
        } else {
            println("$i \t ${df.format(i * 1.609)}")
        }
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/non-plagiarized/14/Soal4.ts
import { DecimalFormat } from 'java.text';

/**
 *
 * @author 4900D0A19B6894A4A514E9FF3AFCC2C0
 */
function main(): void {
    const arr: number[][] = new Array(11).fill(0).map(() => new Array(2));
    const df: DecimalFormat = new DecimalFormat('#.###');
    for (let i = 0; i < 11; i++) {
        if (i === 0) {
            console.log('Miles \t Kilometers');
        } else {
            console.log(`${i}\t${df.format(i * 1.609)}`);
        }
    }
}

main();

